lnwallet/channel.go

```diff
@@ -8366,11 +8366,12 @@ func (lc *LightningChannel) availableCommitmentBalance(view *htlcView,
 	if theirBalance < commitFeeWithHtlc && ourBalance >= nonDustHtlcAmt {
 		// see https://github.com/lightning/bolts/issues/728
 		ourReportedBalance := nonDustHtlcAmt - 1
-		lc.log.Infof("Reducing local balance (from %v to %v): "+
-			"remote side does not have enough funds (%v < %v) to "+
-			"pay for non-dust HTLC in case of unilateral close.",
-			ourBalance, ourReportedBalance, theirBalance,
-			commitFeeWithHtlc)
+		lc.log.Infof("Reducing local (reported) balance "+
```

Comment:
nit: if you configure your editor correctly (add a ruler at 80 characters, use tab size of 8), you see that this message can be slightly more compact by fitting a couple of more words on each line.

Reply:
Thanks! Updated.